srcpkgs/growlight/template

```diff
@@ -0,0 +1,17 @@
+# Template file for 'growlight'
+pkgname=growlight
+version=1.2.18
+revision=1
+build_style=cmake
+configure_args=" -DUSE_PANDOC=OFF"
```

Comment:
why disable? Also, doesn't require whitespace.

Reply:
I was under the impression that it is not available in every arch - seems that I am mistaken :) 
It is only used to "compile" the manpages.

Reply:
It is nocross :(

Reply:
Being nocross isn't a huge issue, since it would only be a `hostmakedepends`, right? But yeah, I think ppc might lack haskell support.

Reply:
Yes, I have build the programms for many archs, w/o problem and added pandoc as hostmakedepends. If haskel is not available on ppc Ι can add pandoc and the configure_args for everything else.